server/src/domain/asset/asset.service.ts

```diff
@@ -389,18 +391,32 @@ export class AssetService {
   async update(authUser: AuthUserDto, id: string, dto: UpdateAssetDto): Promise<AssetResponseDto> {
     await this.access.requirePermission(authUser, Permission.ASSET_UPDATE, id);
 
-    const { description, ...rest } = dto;
+    const { description, creationDate, latitude, longitude, ...rest } = dto;
     if (description !== undefined) {
       await this.assetRepository.upsertExif({ assetId: id, description });
     }
 
+    if (creationDate !== undefined || latitude !== undefined || longitude !== undefined) {
+      if ((latitude !== undefined || longitude !== undefined) && !(latitude !== undefined && longitude !== undefined)) {
+        throw new BadRequestException('Latitude and longitude must be provided together');
+      }
+      const { pathMetadata, newExifData } = await this.getPathAndExif(id, {
+        CreationDate: creationDate,
+        GPSLatitude: latitude,
+        GPSLongitude: longitude,
+      });
+      await this.assetRepository.save({ id, sidecarPath: pathMetadata });
+      await this.assetRepository.writeExif(pathMetadata, newExifData);
```

Comment:
It might be preferable to write the sidecar file first, so that if the write fails there will not be a dead path in the database. (cc @jrasm91: do requests run in a transaction by default? If so, this might not be a concern)

Reply:
Whatever it's better to write the sidecar first, it's done